### ISSUE TITLE
Verbeterde WCAG Schrijfwijzer in private documentatie

### ISSUE THREAD

**Issue by erikkroes** [nl-design-system/documentatie#3021]
Schrijfwijzer op slack bijwerken met alle recente input en inzichten

**Comment by Robbert** [comment]
De schrijfwijzer staat nu in een Slack Canvas:
https://codefornl.slack.com/docs/T68FXPFQV/F08S937D6KZ

Die kan in Markdown in /docs/private/ geplaatst worden, en uitgebreid worden
